sources/osgAnimation/UpdateSkeleton.js

```diff
@@ -54,6 +54,13 @@ define( [
         needToValidate: function () {
             return this._needValidate;
         },
+        compare: function ( x, y ) {
```

Comment:
could you move out this function from the class, it does not require to be a member.
just define it outside like
var compareBone = function( a, b ) {...};
then use it directly